Sources/Defaults/Utilities.swift

```diff
@@ -194,7 +194,7 @@ extension Defaults.Serializable {
 			return anyObject
 		}
 
-		guard let nextType = T.Serializable.self as? any Defaults.Serializable.Type else {
+        guard let nextType = T.Serializable.self as? any Defaults.Serializable.Type, nextType != T.self else {
```

Comment:
Please fix the indention, we are using tab indention.

Reply:
Fixed that. Sorry about that.